pkg/api/v1/validation/validation.go

```diff
@@ -60,6 +60,15 @@ func ValidateResourceRequirements(requirements *v1.ResourceRequirements, fldPath
 		allErrs = append(allErrs, validateContainerResourceName(string(resourceName), fldPath)...)
 		// Validate resource quantity.
 		allErrs = append(allErrs, ValidateResourceQuantityValue(string(resourceName), quantity, fldPath)...)
+
+		// For GPUs, limits has to be set equal to requests.
+		if resourceName == v1.ResourceNvidiaGPU {
```

Comment:
IIRC, for resources like CPU/memory, if their requests are not configured, they will be defaulted to the limits.